src/components/Search/index.tsx

```diff
@@ -120,7 +120,7 @@ function mapTransactionItemToSelectedEntry(item: TransactionListItemType, outsta
             amount: item.modifiedAmount ?? item.amount,
             convertedAmount: item.convertedAmount,
             currency: item.currency,
-            isFromOneTransactionReport: item.isFromOneTransactionReport,
+            isFromOneTransactionReport: item.report?.transactionCount === 1,
```

Comment:
NAB: We can recreate a sharable util to check if a report is `isOneTransactionReport`